Extensions/CartPageExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Infrastructure;
+using Infrastructure.Pages;
+
+namespace Extensions
+{
+    public static class CartPageExtensions
+    {
```

Comment:
Why unit and not product? I know that in the DOM the elements are called units, but maybe it is more indicative to call it here a product? It's up to you, you can maybe ask other people!

Also, maybe you should consider not using an array and perhaps a Tuple? You can use any object that is based on key/value and then instead of accessing an array you can access the prices by the value of the key (for instance, a Tuple that has ('productPrice', productPrice) and another ('totalPrice', totalPrice)